topostats/tracing/ordered_tracing.py

```diff
@@ -637,12 +640,39 @@ def ordered_trace_mask(ordered_coordinates: npt.NDArray, shape: tuple) -> npt.ND
     return ordered_mask
 
 
+def trace_mask(coordinates: npt.NDArray, shape: tuple) -> npt.NDArray:
+    """
+    Obtain a mask of the trace coordinates with each trace pixel.
+
+    Parameters
+    ----------
+    coordinates : npt.NDArray
+        Ordered array of coordinates.
+
+    shape : tuple
+        The shape of the array bounding the coordinates.
+
+    Returns
+    -------
+    npt.NDArray
+        NxM image with each pixel in the ordered trace labeled in ascending order.
+    """
+    mask = np.zeros(shape)
+    if isinstance(coordinates, np.ndarray):
+        mask[coordinates[:, 0], coordinates[:, 1]] = 1
+        return mask
+    if isinstance(coordinates, list):
+        for mol_coords in coordinates:
+            mask[mol_coords[0], mol_coords[1]] = 1
```

Comment:
Same here (and also this fails upon parsing a list of 2D coordinate arrays)

```suggestion
        mask[mol_coords[:, 0], mol_coords[:, 1]] = np.arange(len(coordinates))
```